apps/system/utils/kdbg_irqinfo.c

```diff
@@ -15,19 +15,42 @@
  * language governing permissions and limitations under the License.
  *
  ****************************************************************************/
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <sys/mount.h>
 
-#include <tinyara/config.h>
-#include <tinyara/irq.h>
-#include <sys/types.h>
+#include <tinyara/fs/fs.h>
 
+#include "kdbg_utils.h"
+
+#define IRQ_BUFLEN 64
 
 int kdbg_irqinfo(int argc, char **args)
 {
-#ifdef CONFIG_DEBUG_IRQ_INFO
-	irq_info();
+	int ret;
+	char *filepath;
+	char buf[IRQ_BUFLEN];
+
+	/* Mount Procfs to use */
+	ret = mount(NULL, PROCFS_MOUNT_POINT, PROCFS_FSTYPE, 0, NULL);
```

Comment:
Why do you add this even you modified fs_initialize.c file for fs_auto_mount()?

Reply:
I added for making all use same path when automount is enabled